roles/deploy/tasks/main.yml

```diff
@@ -16,3 +16,10 @@
     notify: 
       - ensure systemd
       - check_http_server_running
+
+  - name: report deployment to influxdb
+    ansible.builtin.uri:
+      url: http://errbit2.obsmgmt.opensuse.org:48086/write?db=SystemHealthMonitoring
+      method: POST
+      body: "deploymentcount value=1i"
```

Comment:
This is nice an all but can we track the size (lines changed or something) and age (time since the last deployment) instead?

Reply:
If you really want to go deep you can also calculate cycle time:  time between pull request opened to pull-request deployed for all the PRs that bring the collection of commits we deploy.

Reply:
I would prefer to keep things simple and not tie ourselves too much to the GitHub APIs to avoid vendor-lockin.

Reply:
We are *already* tracking deployments on github and use it as sync point so unless you have a better idea and got us to change that we are with github deployments :-)